aten/src/ATen/native/cuda/BinaryMulDivKernel.cu

```diff
@@ -45,7 +45,7 @@ struct MulFunctor<bool> {
 
 
 void div_kernel_cuda(TensorIterator& iter) {
-  if (!isIntegralType(iter.common_dtype(), /*includeBool*/ false) && iter.is_cpu_scalar(2)) {
+  if (!isIntegralType(iter.common_dtype(), /*includeBool*/ true) && iter.is_cpu_scalar(2)) {
```

Comment:
does this affect anything?  neither of the dispatch macros has bool?

Reply:
You are right, it doesn't, which makes me a bit confused on what should be the expected behaviour here:
```
$ python -c "import torch;print(torch.tensor([True, False]) / True)"
tensor([1., 0.])
```
Although numpy behaves the same:
```
$ python -c "import numpy as np;print(np.asarray([True, False]) / True)"
[1. 0.]
```